src/Counters.jl

```diff
@@ -1,7 +1,7 @@
 using Base.Iterators: flatten
 
-flops(a, b, size) = prod(ind -> size[ind], a ∪ b, init=1)
-flops(a, b, size, keep) = prod(ind -> size[ind], flatten((a ∪ b, ∩(keep, a, b))), init=1)
+flops(a, b, size) = prod(ind -> size[ind], a ∪ b, init=BigInt(1))
```

Comment:
`BigInt(1)` is ok, but this is slightly better. After compiler optimizations, both should be equivalent but with `one` we can be sure that the parameter is statically known.
```suggestion
flops(a, b, size) = prod(ind -> size[ind], a ∪ b, init=one(BigInt))
```